torchbiggraph/train_cpu.py

```diff
@@ -65,25 +65,24 @@
     tag_logs_with_process_name,
 )
 
-
 logger = logging.getLogger("torchbiggraph")
 dist_logger = logging.LoggerAdapter(logger, {"distributed": True})
 
 
 class Trainer(AbstractBatchProcessor):
     def __init__(
-        self,
-        model_optimizer: Optimizer,
-        loss_fn: AbstractLossFunction,
-        relation_weights: List[float],
+            self,
```

Comment:
I wouldn't worry about the formatting for now, because our internal formatter will reformat it before merge. We can't expect external contributors to get this right without publishing

@rkindi maybe we can add a black style file or something into the repo to make it easier for external contributors? I'm not sure if there's anything like this for the internal Meta style.